dask/dataframe/multi.py

```diff
@@ -232,6 +232,17 @@ def require(divisions, parts, required=None):
 }
 allowed_left = ("inner", "left", "leftsemi", "leftanti")
 allowed_right = ("inner", "right")
+union_categoricals_dispatch = Dispatch("union_categoricals")
+
+
+@union_categoricals_dispatch.register((pd.DataFrame, pd.Series, pd.Index, pd.Categorical))
+def union_categoricals_pandas(to_union, sort_categories=False, ignore_order=False):
+    return pd.api.types.union_categoricals(to_union, sort_categories=sort_categories, ignore_order=ignore_order)
+
+
+def union_categoricals(to_union, sort_categories=False, ignore_order=False):
+    func = union_categoricals_dispatch.dispatch(type(to_union[0]))
+    return func(to_union, sort_categories=sort_categories, ignore_order=ignore_order)
```

Comment:
Would we want to include these in [`dask.dataframe.backends`]( https://github.com/dask/dask/blob/4799c9d1dd9d9af7df1358aea11c4f49f15d3809/dask/dataframe/backends.py )?